reference/extensions/deployers.rst

```diff
@@ -1,5 +1,65 @@
 .. _reference_extensions_deployers:
 
 Deployers
----------
+=========
 
+Deployers are a mechanism to facilitate copying files form one folder, usually the Conan cache, to user folders.
+While Conan provides two built-in ones (``full_deploy`` and ``direct_deploy``), users can easily manage their own
+with ``conan config install``.
+
+Deployers run before generators, and they can change the target folders.
+For example, if the ``--deploy=full_deploy`` deployer runs before ``CMakeDeps``,
+the files generated by ``CMakeDeps`` will point to the local copy in the user folder done by the ``full_deploy`` deployer,
+and not to the Conan cache.
+
+Deployers can be multi-configuration. Running ``conan install . --deploy=full_deploy`` repeatedly for different profiles
+can achieve a fully self-contained project, including all the artifacts, binaries, and build files.
+This project would be completely independent of Conan and no longer require it at all to build.
+
+
+Built-in deployers
+------------------
+
+.. _reference_extensions_deployer_full_deploy:
+
+full_deploy()
+^^^^^^^^^^^^^
+
+Deploys to ``output_folder`` + ``host/dep/0.1/Release/x86_64`` subfolder
```

Comment:
is it always just `host/dep/0.1/Release/x86_64` or is it somehow computed on fly based on settings?

Reply:
Nice catch, I'll clarify that it's based on context, recipe name, build type and arch